test/web-platform-tests/run-wpts.js

```diff
@@ -64,12 +67,19 @@ describe("web-platform-tests", () => {
                              (reason === "fail-with-canvas" && hasCanvas) ||
                              (reason === "needs-node10" && !hasNode10) ||
                              (reason === "needs-node11" && !hasNode11) ||
-                             (reason === "needs-node12" && !hasNode12);
+                             (reason === "needs-node12" && !hasNode12) ||
+                             (reason === "needs-node13" && !hasNode13);
 
           if (matchingPattern && shouldSkip) {
             specify.skip(`[${reason}] ${testFile}`);
           } else if (expectFail) {
-            runSingleWPT(testFilePath, `[expected fail] ${testFile}`, expectFail);
+            let failReason = "";
+            if (reason.startsWith("needs-node")) {
+              failReason = `: ${reason}`;
+            } else if (reason === "fail-with-canvas") {
+              failReason = ": canvas bug";
```

Comment:
Hmm, do you think "canvas bug" is more helpful than "fail-with-canvas"? I like the idea of people reading the results being able to directly map the output to the .yml file contents, so it'd be good to align on one of those two, instead of translating between them. (Or a third alternative.)